sysutils/smart/src/opnsense/mvc/app/models/OPNsense/Smart/Smart.xml

```diff
@@ -0,0 +1,8 @@
+<model>
```

Comment:
Basically this is ready to be merged now so thanks for the work!

One last question: why does this require an (empty) model? can it be omitted?

Reply:
An empty model is a side effect of using the workflow described in the Hello World example. An empty model is a stub if in the future you need to add plugin properties. Unfortunately, I do not know whether to do this, and how to expand this plug-in. Maybe someday it will be necessary, and maybe not. Although I'll remove the empty model now. If necessary, it will always be possible to return it back from the log.

Reply:
Yup, thanks. I'll merge now 👍